rpcd/playbooks/verify-maas.yml

```diff
@@ -52,3 +52,6 @@
       delay: "5"
       when:
         - maas_use_api | bool
+
+  vars:
+    maas_use_api: true
```

Comment:
this force a non-overridable maas_use_api: True, overridable only through a CLI var.
I'd put that into a group var instead.